tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -1980,6 +1980,14 @@ class Foo { }
                 'case_sensitive' => true,
             ],
         ];
+
+        yield 'do not take the braces part in grouped imports into account' => [
+            <<<'PHP'
+                <?php
+                use Foo\{Bar, Baz};
+                use Foo\NotBarNorBar\DeeperClass;
+                PHP,
+        ];
```

Comment:
```suggestion
        yield 'do not take the braces part in grouped imports into account - case A' => [
            <<<'PHP'
                <?php
                use Foo\{Bar, Baz};
                use Foo\NotBarNorBar\DeeperClass;
                PHP,
        ];

                yield 'do not take the braces part in grouped imports into account - case B' => [
            <<<'PHP'
                <?php
                use Foo\NotBarNorBar\DeeperClass;
                use Foo\{Xxx, Yyy};
                PHP,
            <<<'PHP'
                <?php
                use Foo\{Xxx, Yyy};
                use Foo\NotBarNorBar\DeeperClass;
                PHP,
        ];
```

Reply:
Let's just have one, but more complicated test case, names `case A`/`case B` do not help to understand what is the difference between them.